README.md

```diff
@@ -146,6 +147,9 @@ _Access your assets anywhere._
 
 _Applications, games, resources and other things built and/or maintained by CryptoNote developers._
 
--   [DecentraChat](https://github.com/lithyriolu/decentrachat) - [LithyRiolu](https://github.com/lithyriolu) - [The Lithe Project™](https://github.com/lithe-project)
+-   [DecentraChat](https://github.com/lithyriolu/decentrachat)
+-   [LithyRiolu](https://github.com/lithyriolu)
+-   [The Lithe Project™](https://github.com/lithe-project)
```

Comment:
Can you fix this line to how it was please?

https://github.com/Lithe-Project/Awesome-CryptoNote/pull/2#discussion_r329316698 explains why it was the way it was.